### DISCUSSION TITLE
Add volatile entity

### DISCUSSION THREAD

**Discussion by AntoD3v** [PaperMC/Paper#10717] [Ideas]
### Is your feature request related to a problem?

From time to time, we use volatile entities, which are not intended to remain on the server even after a reboot. (Display Text, etc...)
It can also be useful for server crashes to no longer have this object persistent.

### Describe the solution you'd like.

I propose to create an option in the entity allowing or not to deactivate its backup in the world. 
entity.setVolatile(true)

### Describe alternatives you've considered.

No real idea of alternation

**Comment by TonytheMacaroni** [comment]
Isn't this just https://jd.papermc.io/paper/1.20.6/org/bukkit/entity/Entity.html#setPersistent(boolean)?

**Reply by AntoD3v** [reply]
Ok, I'm sorry, I've looked everywhere but not this method. :/
Thank you for your response @TonytheMacaroni 
I'll close the discussion.
